go/apps/dialogue/tests/test_vumi_app.js

```diff
@@ -318,5 +320,49 @@ describe("app", function() {
                     });
             });
         });
+
+        describe("when the user enters a send state which is a http_json type", function() {
+            it("and it is a GET type the response should be 200", function() {
+                return tester
+                    .setup(function(api) {
+                        api.contacts.add({
+                           msisdn: '+27123'
+                        });
+                        api.http.fixtures.add({request: {method: "GET", url: "www.foo.bar"}});
+                    })
+                    .setup.user.addr('+27123')
+                    .setup.user.state('choice-1')
+                    .input('4')
+                    .check(function(api) {
+                        assert.equal(api.http.fixtures.fixtures[0].responses[0].code, 200);
+                    })
+                    .run();
+            });
+            
+            it("and it is a POST type the response should be 200", function() {
```

Comment:
I'm not sure if this test description makes sense. I think this test is testing that we send the right data to the api. Could we change the test description to mention that? Similarly for the `GET` case.